src/sider/historikk/Historikk.tsx

```diff
@@ -54,6 +54,8 @@ function tagFromKilde(kilde: HistorikkEventType): ReactElement {
       return <Tag variant="info">Friskmelding til arbeidsformidling</Tag>;
     case "DIALOG_MED_BEHANDLER":
       return <Tag variant="neutral">Dialog med behandler</Tag>;
+    case "SEN_OPPFOLGING":
+      return <Tag variant="alt2">Sen oppfølging</Tag>;
```

Comment:
Jeg tror vi i Modia har kalt alt for `Snart slutt på sykepengene`, så `sen fase` er bare et konsept internt hos oss. (Noe vi egentlig burde slutte med i så fall, hvis vi ikke skal bruke det i løsningene våre 🤔). Kan hende Stine har noen tanker her.

Reply:
Ja, disse begrepene har blitt brukt om hverandre på teamet og jeg visste faktisk ikke at det var forskjellen. Betyr det at det gir mening at jeg kun i så fall endrer det som står her til `Snart slutt på sykepengene`? mao. det brukeren ser i appen.